src/scss/components/mobile-nav/mobile-nav.stories.ts

```diff
@@ -0,0 +1,41 @@
+import { html } from "lit";
+
+import type { Meta, StoryObj } from "@storybook/web-components";
+import { Close as MenuToggle } from "../menu-toggle/menu-toggle.stories";
+
+const toolItems = ["Tool Home", "About", "Data Dashboards", "Custom Data Download"];
+const generalItems = ["IATI home", "News", "Events", "Contact", "Help docs"];
+
+
+const meta: Meta = {
+  title: "Components/Mobile Nav",
+  argTypes: {
+    open: {
+      defaultValue: true,
+      control: { type: 'boolean' }
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj;
+
+export const MobileNav: Story = {
+  render: (args) => html`
+    <div class="iati-mobile-nav js-iati-mobile-nav">
```

Comment:
Is it necessary to have `.js-iati-mobile-nav`, or can the javasacript just target `.iati-mobile-nav`?

Reply:
@tillywoodfield there's an opinion that the selectors you used for `querySelector` etc. should not be same as those used for styles (for the sake of resilience\decoupling) but this is just an opinion so happy to change if that is what you would prefer?

Reply:
That makes sense, I think it's fine like this then